docs/contrib/development.md

```diff
@@ -0,0 +1,82 @@
+# Development Guide
+
+Since K3s is written in Go, it is fair to say that the Go tools are all one needs to contribute to this project. Unfortunately, there is a point where this statement no longer holds true when it is required to test or build locally our changes. This document elaborates on the required tooling for K3s development.
```

Comment:
```suggestion
Since K3s is written in Go, it is fair to assume that the Go tools are all one needs to contribute to this project. Unfortunately, there is a point where this no longer holds true when required to test or build local changes. This document elaborates on the required tooling for K3s development.
```